src/containerapp/azext_containerapp/_params.py

```diff
@@ -370,6 +370,16 @@ def load_arguments(self, _):
         c.argument('max_replicas', type=int, help="Maximum number of replicas to run for the Java component.")
         c.argument('route_yaml', options_list=['--route-yaml', '--yaml'], help="Path to a .yaml file with the configuration of a Spring Cloud Gateway route. For an example, see https://aka.ms/gateway-for-spring-routes-yaml")
 
+    with self.argument_context('containerapp env maintenanceconfig add') as c:
+        c.argument('weekday', options_list=['--weekday', '-w'], help="The weekday to schedule the maintenance configuration.")
```

Comment:
The --weekday type is enum, right? Should we add `arg_type=get_enum_type(["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday" ])` ?